ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -44,7 +47,8 @@
         <AccountInfo
           v-if="accountInfo"
           class="mt-15"
-          title="Submtting Party for this Registration"
+          title="Submitting Party for this Registration"
+          desc = "Registration verification statement and decals will be mailed to this address."
```

Comment:
Maybe I should've been more explicit in my previous comment. Remove spaces around the `=` sign for consistency.

Reply:
sorry my bad, i forgot to delete the spaces.